src/balancerd/Cargo.toml

```diff
@@ -44,6 +45,7 @@ workspace-hack = { version = "0.0.0", path = "../workspace-hack" }
 mz-environmentd = { path = "../environmentd", features = ["test"] }
 mz-frontegg-mock = { path = "../frontegg-mock" }
 postgres = "0.19.5"
+reqwest = "0.11.24"
```

Comment:
This is a different version than the rest of our codebase (we need the tlsinfo method that's only present in versions after whatever we have everywhere else). Not sure if this'll cause a lint error.

Reply:
It'll just bump everyone up to 0.11.24, because Cargo considers it to be semver compatible with the version listed elsewhere.